app/assets/javascripts/discourse/app/index.html

```diff
@@ -27,6 +27,15 @@
     <div id='offscreen-content'>
     </div>
 
+    <script>
```

Comment:
This triggers my in-brain CSP 😂 but I guess if it's just for the dev env it's ok 😉

Reply:
That's a good point, I could split it off in the future. But it should work for quite some time.

Reply:
Here we go: https://github.com/discourse/discourse/pull/12401